src/flex.skl

```diff
@@ -618,6 +619,7 @@ void yyfree ( void * M4_YY_PROTO_LAST_ARG );
 
 m4_ifdef( [[M4_YY_NOT_IN_HEADER]],
 [[
+
 #define yy_new_buffer yy_create_buffer
 #define yy_set_interactive(is_interactive) \
```

Comment:
When you are implementing yyset_interactive, could you also remove this yy_set_interactive macro, which could cause confusion.

Reply:
@Explorer09 Good call, I will get this updated tonight.

Reply:
@Explorer09 This is done.